akka-docs/src/main/paradox/stream-typed.md

```diff
@@ -1 +1,49 @@
-# Streams
\ No newline at end of file
+# Typed Actors and Streams
+
+@@@ warning
+
+This module is currently marked as @ref:[may change](common/may-change.md) in the sense
+  of being the subject of active research. This means that API or semantics can
+  change without warning or deprecation period and it is not recommended to use
+  this module in production just yet—you have been warned.
+  
+@@@
+
+[Akka Streams](stream/index.md) make it easy to model typesafe message processing pipelines. With typed actors it is possible to extend these pipelines to actors without loosing the type information.
+
```

Comment:
Perhaps also add "Please note that while the `ActorMaterializer` in this module takes and works with a typed `ActorSystem`, all the other types of Akka Streams remain the same, so there is seamless inter op between akka-stream and akka-stream-typed. This page explains the inter op between Streams and Typed Actors"??